package.json

```diff
@@ -155,7 +153,6 @@
     },
     "husky": {
         "hooks": {
-            "pre-commit": "run-p changelog schema schema-higlass schema-theme schema-template format && git add .",
             "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
```

Comment:
Also, since we use squash + merging, idk if it makes sense to have this hook either. Instead, something like a GitHub action to lint the _PR title_ would probably make more sense: https://github.com/vitejs/vite/blob/main/.github/workflows/semantic-pull-request.yml

Not sure if either of you have thoughts on this: @etowahadams @sehilyi

Reply:
It would make sense to use the `semantic-pull-request` you shared and remove the `commitlint` we use for every commit.

We were using [another semantic PR](https://github.com/zeke/semantic-pull-requests) which is deprecated and does not work anymore. We were using `commitlit` to handle a case where there is only one commit in a PR because in this case [GitHub used the commit message and ignored the PR title](https://github.com/refined-github/refined-github/issues/276). But, [using PR title by default is now supported](https://github.blog/changelog/2022-05-11-default-to-pr-titles-for-squash-merge-commit-messages/), so I don't think we need `commitlint` anymore.

Reply:
I agree! No need to have every conventional commit name be linted.